content/guides/destructuring.adoc

```diff
@@ -137,9 +148,18 @@ You can ignore bindings that you don't intend on using by binding them to any sy
 ;= Odd names: Michael Aaron Earl
 ----
 
-The convention for this is to use an underscore like above.
+The convention for this is to use an underscore as above. 
 
-You can use `:as all` to bind the entire vector to the symbol `all`.
+There is nothing special about the underscore. Being a symbol like any other, it retains value of its last binding:
```

Comment:
First sentence is good, latter and example is unimportant and distracting. Link to the FAQ entry if needed. https://clojure.org/guides/faq#underscore

Reply:
Yes!